blazor/ClientSideExample/ClientSideExample/ViewModels/CounterViewModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Reactive;
+using System.Threading.Tasks;
+using ReactiveUI;
+
+namespace ClientSideExample.ViewModels
+{
+    public class CounterViewModel : ReactiveObject
+    {
+        private int currentCount;
+
+        public CounterViewModel()
+        {
+            Increment = ReactiveCommand.CreateFromTask(IncrementCount);
+        }
+
+        public int CurrentCount => currentCount;
+        
+
+        public ReactiveCommand<Unit, Unit> Increment;
+
+        private Task IncrementCount()
+        {
+            currentCount++;
+            return Task.CompletedTask;
```

Comment:
Make this return Task<int> and return Task.FromResult(currentCount)

Reply:
okay.

Reply:
Done.